sdk/storage/Azure.Storage.DataMovement/src/JobPlan/JobPartPlanFile.cs

```diff
@@ -68,6 +62,7 @@ public static async Task<JobPartPlanFile> CreateJobPartPlanFileAsync(
 
             using (FileStream fileStream = File.Create(result.FileName.ToString()))
             {
+                CancellationHelper.ThrowIfCancellationRequested(cancellationToken);
                 await headerStream.CopyToAsync(fileStream).ConfigureAwait(false);
```

Comment:
`CopyToAsync` accepts a cancellationtoken

Reply:
Looking at the Streams source code it does seem to be 8KB